content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: 'setTimeout()'
+Description: 'setTimeout() allows users to execute callbacks after a period of time expressed in milliseconds'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+`setTimeout()` is a function serviced globally by the window object provided by the user's browser, it is not part of JavaScript because it is not a JavaScript function.
+
+## Syntax
+
+`setTimeout()` is capable of receiving multiple parameters where the first is a callback function, the second parameter receives a number that represents the time in milliseconds **(1s = 1000ms)**, that defines the time needed for the callback to execute, the third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
+
+The provided sample code contains some examples of how setTimeout is composed:
+
+```pseudo
+setTimeout(callback)
+
+setTimeout(callback, delay)
+
+setTimeout(callback, delay, param)
+```
+
+## Usage example
+
+The following pieces of code will show some examples about the setTimeout implementation:
+
+```shell
+// Defining the function
+
+function sayHello() {
+  console.log('Hello from Codecademy');
+}
+
+// Calling the function using setTimeout.
+
+setTimeout(sayHello, 4000);
+```
+
+The output will be displayed after 4 seconds:
+
+```output
+Hello from Codecademy
+```
+
+An example using [arrow functions](https://www.codecademy.com/resources/docs/javascript/arrow-functions) would be:
```

Comment:
This would be a header named example using arrow functions. In which case I recommend having 1-2 sentences explaining the code below. Within those sentences, you can include the arrow functions entry link.